searx/static/themes/simple/src/less/style.less

```diff
@@ -281,24 +281,12 @@ article[data-vim-selected].category-social {
     color: var(--color-result-description-highlight-font);
   }
 
-  img {
-    &.thumbnail {
-      .ltr-float-left();
-      padding-top: 0.6rem;
-      .ltr-padding-right(1rem);
-      width: 20rem;
-      height: unset; // remove heigth value that was needed for lazy loading
-    }
-
-    &.image {
-      .ltr-float-left();
-      padding-top: 0.6rem;
-      .ltr-padding-right(1rem);
-      width: 7rem;
-      max-height: 7rem;
-      object-fit: scale-down;
-      object-position: right top;
-    }
+  img.thumbnail {
+    .ltr-float-left();
```

Comment:
See my first comment on the PR.

Reply:
> This is slightly out of topic: more than once, it was asked to put the image of the right rather than of the left (at least for LTR languages). With images on the right, all the tittles are aligned on the left side --> it is easier to read them.

In the current layout (in current master) the thumbnails are on the left side (at least for LTR languages) / here is a screenshot from a `default.html` result, how it is rendered in branch mast:

![grafik](https://github.com/searxng/searxng/assets/554536/9bd94c16-6bed-4f9a-9bf9-0a0a4ee01c5a)

We can change the layout in a follow up PR / as you said "slightly out of topic" in this PR to change the layout (substantially).